spec/system/provider_interface/make_offer_ske_standard_flow_feature_flag_enabled_spec.rb

```diff
@@ -0,0 +1,359 @@
+require 'rails_helper'
+
+RSpec.feature 'Provider makes an offer with SKE enabled' do
+  include DfESignInHelpers
+  include ProviderUserPermissionsHelper
+
+  let(:provider_user) { create(:provider_user, :with_dfe_sign_in) }
+  let(:provider) { provider_user.providers.first }
+  let(:ratifying_provider) { create(:provider) }
+  let(:application_form) { build(:application_form, :minimum_info) }
+  let!(:application_choice) do
+    create(:application_choice, :awaiting_provider_decision,
+           application_form:,
+           course_option:)
+  end
+  let(:course) do
+    build(:course, :full_time, provider:, accredited_provider: ratifying_provider)
+  end
+  let(:course_option) { build(:course_option, course:) }
+
+  scenario 'Making an offer for the requested course option' do
```

Comment:
That's a good point! Since I joined I wanna do smaller scenarios but all system specs have huge one scenarios only (no criticism as people could have a point about it in the past) - I think if we do that right now the codebase will start to look different, with different standards. Maybe we can talk about changing this standard at some point?

Reply:
well, i'm pretty sure this is the biggest scenario yet, in a codebase full of big scenarios. it's long enough that i don't have the patience to read the whole thing :-) perhaps the issue is doing _all_ the testing in the system spec - could some of the validation side of things be tested as a unit test of `forms/provider_interface/offer_wizard.rb`?